controllers/csiscaleoperator_controller.go

```diff
@@ -324,40 +338,48 @@ func (r *CSIScaleOperatorReconciler) getAccessorAndFinalizerName() (metav1.Objec
 	return accessor, finalizerName, nil
 }
 
-func (r *CSIScaleOperatorReconciler) deleteClusterRolesAndBindings() error {
-	/*	if err := r.deleteClusterRoleBindings(instance); err != nil {
+func (r *CSIScaleOperatorReconciler) deleteClusterRolesAndBindings(instance *csiscaleoperator.CSIScaleOperator) error {
+	logger := csiLog.WithName("in deleteClusterRolesAndBindings")
+
+	logger.Info("calling deleteClusterRoleBindings")
+	if err := r.deleteClusterRoleBindings(instance); err != nil {
 		return err
-	}*/
+	}
 
-	if err := r.deleteClusterRoles(); err != nil {
+	logger.Info("calling deleteClusterRoles")
+	if err := r.deleteClusterRoles(instance); err != nil {
 		return err
 	}
 	return nil
 }
 
-func (r *CSIScaleOperatorReconciler) deleteClusterRoles() error {
+func (r *CSIScaleOperatorReconciler) deleteClusterRoles(instance *csiscaleoperator.CSIScaleOperator) error {
+	logger := csiLog.WithName("in deleteClusterRoles")
 
-	/*	clusterRoles := r.getClusterRoles(instance)
+	logger.Info("entering deleteClusterRoles")
```

Comment:
`csi.WithName("function name")` will help to identify that the log belongs to which method, so no need for another entry message here. We may add logs like `deleting cluster roles` but I guess entry message is not required.

Reply:
Done.

Reply:
Thanks for fixing this, can you do the same changes in these methods too `deleteClusterRoleBindings`  `deleteCSIDriver`

Reply:
Done